File: CPP/beginner/1012.cpp
#include <stdio.h>
#include <math.h> 

int main() {
 
  double pi = 3.14159, A = 0, B = 0, C = 0, Area = 0;

  scanf("%lf %lf %lf", &A, &B, &C);
  
  Area = (A * C) / 2;
   printf("TRIANGULO: %.3lf\n", Area);
  
  Area = pi * pow(C, 2); 
  printf("CIRCULO: %.3lf\n", Area);

  Area = ((A + B) * C) / 2;
   printf("TRAPEZIO: %.3lf\n", Area);

  Area = pow (B, 2);
   printf("QUADRADO: %.3lf\n", Area);
  
  Area = A * B;
   printf("RETANGULO: %.3lf\n", Area);

    return 0;
}

File: CSharp/beginner/1012.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        string[] area = Console.ReadLine().Split(' ');
        double a = double.Parse(area[0]);
        double b = double.Parse(area[1]);
        double c = double.Parse(area[2]);

        double triangulo = (a * c) / 2;
        double circulo = 3.14159 * Math.Pow(c, 2);
        double trapezio = ((a + b) * c) / 2;
        double quadrado = Math.Pow(b, 2);
        double retangulo = a * b;

        Console.WriteLine($"TRIANGULO: {triangulo:F3}");
        Console.WriteLine($"CIRCULO: {circulo:F3}");
        Console.WriteLine($"TRAPEZIO: {trapezio:F3}");
        Console.WriteLine($"QUADRADO: {quadrado:F3}");
        Console.WriteLine($"RETANGULO: {retangulo:F3}");
    }
}
